src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,124 @@
+import { ChangeEvent, FC, memo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
```

Comment:
It is better not to shorten the names to unreadable ones (`cn` -> `classNames`).
This applies to all code.